content/page/events.en.md

```diff
@@ -8,7 +8,7 @@ draft: false
 Exodus Privacy is regulary performing talks about its works and results, in French or in English. If you want to invite us, feel free to [contact us](/en/page/who/).
 
 ## Next events
-* [FR] [Entrée libre](https://entree-libre.frama.site/), Quimper (France), 27th, 28th and 29th August 2019
+* [FR] [Entrée libre](https://www.centredesabeilles.fr/entree-libre/), Quimper (France), 27th, 28th and 29th August 2019, talk the 27th of August at 4 pm
```

Comment:
Should we just write `Quimper (France), 27th August 2019 (4pm)` or something like this?

Reply:
Yes, we should, you are right :)